src/juju_doctor/builtin/application-exists.py

```diff
@@ -0,0 +1,89 @@
+"""Application-exists verbatim builtin plugin."""
+
+import logging
+from typing import Dict, Optional
+
+from pydantic import BaseModel, ConfigDict, Field
+from rich.logging import RichHandler
+
+from juju_doctor.artifacts import read_file
+
+logging.basicConfig(level=logging.WARN, handlers=[RichHandler()])
```

Comment:
Iiuc, this detail (call to basicConfig) should be done in juju doctor, not in the builtins themselves.